views/templates/layout.html.twig

```diff
@@ -16,17 +16,17 @@
 		<nav class = "p-3 bg-secondary">
 			<ul class="nav d-flex justify-content-end">
 				<li class="nav-item">
-					<a href="/custom_cms/home" class="nav-link active text-white">
+					<a href="/cms/home" class="nav-link active text-white">
 						Home
 					</a>
 				</li>
 				<li>    
-					<a href="/custom_cms/contact" class="nav-link text-white">
+					<a href="/cms/contact" class="nav-link text-white">
 						Contacts
 					</a>
 				</li>
 				<li>    
-					<a href="/custom_cms/block" class="nav-link text-white">
+					<a href="/cms/block" class="nav-link text-white">
```

Comment:
Can we remove our dependence on the basepath  and instead just keep the actual URL ?